Lib/test/test_fstring.py

```diff
@@ -1831,6 +1831,42 @@ def test_newlines_in_format_specifiers(self):
         for case in valid_cases:
             compile(case, "<string>", "exec")
 
+    def test_raw_fstring_format_spec(self):
+        """Test raw f-string format spec behavior (Issue #137314).
+
+        Raw f-strings should preserve literal backslashes in format specifications,
+        not interpret them as escape sequences.
+        """
```

Comment:
```suggestion
        # Test raw f-string format spec behavior (Issue #137314).
        #
        # Raw f-strings should preserve literal backslashes in format specifications,
        # not interpret them as escape sequences.
```

Docstrings are rendered upon failures.